addons/sys_core/stringtable.xml

```diff
@@ -55,7 +56,7 @@
             <French>Démuter les clients</French>
             <Japanese>クライアントの消音解除</Japanese>
             <Czech>Zrušit ztlumení diváků</Czech>
-            <German>Stummschaltung der Clients aufheben</German>
+            <German>Clientstummschaltung aufheben</German>
```

Comment:
Hard to read. And it's not clear that all clients are unmuted.

Reply:
reverted